tests/test_browser.py

```diff
@@ -3361,6 +3366,30 @@ def test_modularize_network_error(self):
     os.remove('a.out.wasm')
     self.run_browser('a.html', '...', '/report_result?abort(both async and sync fetching of the wasm failed)')
 
+  def test_modularize_init_error(self):
+    src = open(path_from_root('tests', 'browser_test_init_crash.cpp')).read()
+    create_test_file('test.cpp', src)
+    browser_reporting_js_path = path_from_root('tests', 'browser_reporting.js')
+    self.compile_btest(['test.cpp', '-s', 'MODULARIZE=1', '-s', 'EXPORT_NAME="createModule"', '--extern-pre-js', browser_reporting_js_path])
```

Comment:
`compile_btest` can receive the path to the test, avoiding the need to define `src` earlier.